config/webpack/webpack.client.babel.ts

```diff
@@ -136,6 +138,7 @@ export default {
   entry: path.join(moduleRoot, `src/index.polyfilled.ts`),
 
   output: {
+    globalObject: 'self',
```

Comment:
Webworkers have now window context, so we need to declare the self context here in order for hot reloading (HMR/react-refresh) to work correctly.